src/AccessibilityInsights/MainWindowHelpers/Initialize.cs

```diff
@@ -201,7 +201,8 @@ private static void InitSelectActionMode()
             sa.IsFocusSelectOn = ConfigurationManager.GetDefaultInstance().AppConfig.SelectionByFocus;
             sa.IsMouseSelectOn = ConfigurationManager.GetDefaultInstance().AppConfig.SelectionByMouse;
             sa.TreeViewMode = ConfigurationManager.GetDefaultInstance().AppConfig.TreeViewMode;
-
+            // TODO: Enable this once Axe.Windows is updated
+            //SelectAction.ShouldTestAllChromiumContent = ConfigurationManager.GetDefaultInstance().AppConfig.ShouldTestAllChromiumContent;
```

Comment:
Axe.Windows has indeed been updated

Reply:
Oops! Had that change locally but hadn't pushed it.